docs/guides/building/macos-bundle.md

```diff
@@ -0,0 +1,66 @@
+- **Bundle icon:** Your app's icon. Uses the first `.icns` file listed in the [`tauri.bundle.icon`] array.
+- **Minimum system version:** Configured by the [`tauri.bundle.macOS.minimumSystemVersion`] property.
+- **DMG license file:** A license that is added to the `.dmg` file. Configure by the [`tauri.bundle.macOS.license`] property.
+- **[Entitlements.plist file]:** Entitlements control what APIs your app will have access to. Configured by the [`tauri.bundle.macOS.entitlements`] property.
+- **Exception domain:** an insecure domain that your application can access such as a `localhost` or a remote `http` domain. It is a convenience configuration around `NSAppTransportSecurity > NSExceptionDomains` setting `NSExceptionAllowsInsecureHTTPLoads` and `NSIncludesSubdomains` to true. See [`tauri.bundle.macOS.exceptionDomain`] for more information.
+- **Bootstrapper** Instead of launching the app directly, you can configure the bundled app to run a script that tries to expose the environment variables to the app; without that, you'll have trouble using system programs because the `PATH` environment variable isn't correctly. Enable it with [`tauri.bundle.macOS.useBootstrapper`].
+
+:::info
+These options generate the application bundle [Info.plist file]. You can extend the generated file with your own `Info.plist` file stored on the Tauri folder (`src-tauri` by default). The CLI merges both `.plist` files on production, and the core layer embeds it on the binary on development.
+:::
+
+[application bundle]: https://developer.apple.com/library/archive/documentation/CoreFoundation/Conceptual/CFBundles/BundleTypes/BundleTypes.html
+[`tauri.bundle.macos.minimumsystemversion`]: ../../api/config#macconfig.minimumsystemversion
+[lsminimumsystemversion]: https://developer.apple.com/documentation/bundleresources/information_property_list/lsminimumsystemversion
+[apple silicon macs]: https://support.apple.com/en-us/HT211814
+[universal macos binary]: https://developer.apple.com/documentation/apple-silicon/building-a-universal-macos-binary
+[rosetta]: https://support.apple.com/en-gb/HT211861
+[macos app categories]: https://developer.apple.com/app-store/categories/
+[`package.productname`]: ../../api/config#packageconfig.productname
+[`package.version`]: ../../api/config#packageconfig.version
+[`tauri.bundle.category`]: ../../api/config#bundleconfig.category
+[`tauri.bundle.copyright`]: ../../api/config#bundleconfig.copyright
+[`tauri.bundle.icon`]: ../../api/config#bundleconfig.icon
+[`tauri.bundle.macos.license`]: ../../api/config#bundleconfig.icon
+[entitlements.plist file]: https://developer.apple.com/documentation/bundleresources/entitlements
+[`tauri.bundle.macos.entitlements`]: ../../api/config#macconfig.entitlements
+[`tauri.bundle.macos.exceptiondomain`]: ../../api/config#macconfig.exceptiondomain
+[`tauri.bundle.macos.usebootstrapper`]: ../../api/config#deb.usebootstrapper
```

Comment:
This doesn't currently exist in the config(?)

Reply:
useBootstrapper was removed a few weeks ago

Reply:
Okay, I'll remove that from here then